pkg/infoschema/infoschema_test.go

```diff
@@ -925,6 +925,17 @@ func (tc *infoschemaTestContext) runDropSchema() {
 	})
 }
 
+func (tc *infoschemaTestContext) runRecoverSchema() {
```

Comment:
not used?

Reply:
https://github.com/pingcap/tidb/pull/51760/files#diff-a32d2684983f76c2e21aa607d1fb933c3b269883f1a19ec611d90bfb53141252R1119  here